src/uct/ib/mlx5/dv/ib_mlx5_ifc.h

```diff
@@ -96,6 +97,18 @@ enum {
     UCT_IB_MLX5_CAP_2_GENERAL = 0x20,
 };
 
+enum {
+    UCT_IB_MLX5_HCA_CAPS_2_CROSS_VHCA_OBJ_TO_OBJ_LOCAL_MKEY_TO_REMOTE_MKEY = 0x100
+};
+
+enum {
+    UCT_IB_MLX5_HCA_CAPS_2_ALLOWED_OBJ_FOR_OTHER_VHCA_ACCESS_MKEY = 0x4
+};
+
```

Comment:
it can be the same enum

Reply:
@yosefe wdyt? shall we combine them to a single enum?

Reply:
IMO it can be same enum

Reply:
done